plaso/storage/sqlite/merge_reader.py

```diff
@@ -98,80 +91,67 @@ def __init__(self, storage_writer, path):
 
       self._add_container_type_methods[container_type] = method
 
+  # pylint: disable=unused-argument
```

Comment:
This and other code changes turns off all the re-serialization code - why are you doing this? If you want to remove the functionality, please remove it completely from the interface etc, otherwise please restore the feature.

Reply:
Let's chat more about this. For now I've done it for testing and experimenting with serializing to SQLite native types.